scripts/generate_pinval/constants.py

```diff
@@ -1,5 +1,13 @@
 """Constants that are shared between scripts in this module"""
 
 # Temporary solution for run_id mapping, a problem that occurs when the model run_id
-# differs between the model values and the comps
-RUN_ID_MAP = {"2025-02-11-charming-eric": "2025-04-25-fancy-free-billy"}
+# differs between the model values and the comps.
+# Currently we don't need more than one run because comps and cards are using the same
+# final run ID, but we're keeping this map around with the expectation that we may
+# need it for a more permanent fix
+RUN_ID_MAP = {"2025-02-11-charming-eric": "2025-02-11-charming-eric"}
```

Comment:
See here for an explanation of this (confusing) change, and why I'm not bothering cleaning it up just yet: https://github.com/ccao-data/data-architecture/pull/856#discussion_r2198852995